src/cayenne/data/deposit.clj

```diff
@@ -46,7 +46,7 @@
           (assoc :citation-count (-> with-length-doc :citations count))
           (assoc :matched-citation-count (->> with-length-doc
                                               :citations
-                                              (filter #(-> % :match nil? not))
+                                              (keep :match)
```

Comment:
Per other use of `keep`, I'm not convinced that this has the required behaviour?

Reply:
In this scenario I don't think the behaviour is different since the result is just threaded to `count`, however, I think `filter` is more idiomatic.